src/auth/loginModule.ts

```diff
@@ -1,13 +1,13 @@
 import { Elysia } from "elysia";
-import { User, verifyUser, registerUser, LOGIN_FAILURE_REASON } from "./auth";
+import { User, verifyUser, registerUser, LOGIN_FAILURE_REASON } from "./authModule";
 import { Signup } from "../pages/Signup";
 import { Login } from "../pages/Login";
 import cookie from "@elysiajs/cookie";
 
 // TODO: Make config
 const COOKIE_MAX_AGE = 7 * 86400;
 
-export const login = new Elysia()
+export const loginModule = new Elysia({ name: "loginModule" })
```

Comment:
Renaming these because while creating a login plugin, I was running into issues with a module being called `profile.ts` and the page being called `Profile.tsx` and  importing `profile` would import `profile.tsx` which doesn't exist causing complication issues. Adding a `Module` suffix fixes the name collision.

Reply:
I am open to better naming of these but will leave it as this for now unless someone else has better ideas.